llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -389,6 +391,9 @@ struct VPTransformState {
 
   /// VPlan-based type analysis.
   VPTypeAnalysis TypeAnalysis;
+
+  /// VPlan-based dominator tree.
+  VPDominatorTree *VPDT = nullptr;
```

Comment:
@fhahn  attempted to use VPDominatorTree instead of VPDominatorTree *. However, there's a circular dependency between VPlan.h and VPlanDominatorTree.h. I've been trying to break the dependency for a few days but haven't had much success. Do you have any thoughts?

Reply:
Hmm, let me check

Reply:
@fhahn Any news on this? Or could we proceed with using the VPDominatorTree pointer?